Ginger/Ginger/UserControlsLib/TextEditor/Gherkin/GherkinPage.xaml.cs

```diff
@@ -696,7 +696,7 @@ public bool Load(string FileName)
             }
             
             ARP = new ActivitiesRepositoryPage(WorkSpace.Instance.SolutionRepository.GetRepositoryItemRootFolder<Activity>(), null, mSolTags, ArrowButtonHandler);
-            ARP.xActivitiesRepositoryGrid.EnableTagsPanel = false;
+            //ARP.xActivitiesRepositoryListView.EnableTagsPanel = false;
```

Comment:
Need to add EnableTagsPanel to ListView.xaml.cs and allow this row for not breaking existing functionalities